src/main/java/DukeException.java

```diff
@@ -0,0 +1,14 @@
+/**
+ * Class to represent all the exceptions met by Duke.
+ */
+public class DukeException extends Exception {
```

Comment:
Extracting the various type of exceptions into different classes and inheriting from DukeException could be a possible suggestion to make it more OOP.